PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/ui/SignInWindow.java

```diff
@@ -139,15 +131,121 @@ public void actionPerformed(ActionEvent e) {
         ButtonGroup buttonGroup = new ButtonGroup();
         buttonGroup.add(deviceLoginRadioButton);
         buttonGroup.add(automatedRadioButton);
+        buttonGroup.add(azureCliRadioButton);
         deviceLoginRadioButton.setSelected(true);
-        refreshAuthControlElements();
 
         init();
     }
 
+    public AuthMethodDetails getAuthMethodDetails() {
+        return authMethodDetailsResult;
+    }
+
+    public static SignInWindow go(AuthMethodDetails authMethodDetails, Project project) {
+        SignInWindow d = new SignInWindow(authMethodDetails, project);
+        d.show();
+        if (d.getExitCode() == DialogWrapper.OK_EXIT_CODE) {
+            return d;
+        }
+
+        return null;
+    }
+
+    @Override
+    public void doCancelAction() {
+        authMethodDetailsResult = authMethodDetails;
+        super.doCancelAction();
+    }
+
+    @Override
+    public void doHelpAction() {
+        JXHyperlink helpLink = new JXHyperlink();
+        helpLink.setURI(URI.create("https://docs.microsoft.com/en-us/azure/azure-toolkit-for-intellij-sign-in-instructions"));
```

Comment:
How about using short url aka.ms/... to redirect?

Reply:
Will contact our PM and do this in later PR~